inst/rmarkdown/templates/qc_report/skeleton/skeleton.Rmd

```diff
@@ -61,7 +61,7 @@ params:
     value: null
     input: numeric
   pca_assay_name:
-    label: Assay with normalized counts to use (`cpm`, `tpm`, `rpkm`, or `voom`)
+    label: Assay with normalized counts to use (cpm, tpm, rpkm, or voom)
```

Comment:
@danielinteractive should it be as it is or should we have ticks in here for abbreviations?

Reply:
In this case arguments (and not methods) are described, therefore ticks make sense